RadixWallet/Features/DebugInspectProfile/InspectProfile+View.swift

```diff
@@ -181,6 +181,7 @@ extension DebugInspectFactorSourceView {
 				DeviceFactorSouceView(deviceFactorSource: deviceFactorSource)
 			case let .ledger(ledgerFactorSource):
 				LedgerFactorSourceView(ledgerFactorSource: ledgerFactorSource)
+			default: fatalError("DISCREPANCY: Found non .device | .ledger factor source. A real world user cannot possible have this.")
```

Comment:
This is safe for our users, it is not possible for our users to create any other factor source than `DeviceFactorSource` or `LedgerFactorSource`.